common/thorhelper/thorcommon.cpp

```diff
@@ -42,6 +42,8 @@
 
 #define ROWAGG_PERROWOVERHEAD (sizeof(AggregateRowBuilder))
 
+const cycle_t defaultFlushTimerInterval = nanosec_to_cycle(1000000000); // Cycles
```

Comment:
>accessing defaultFlushTimerInterval is potentially problematic - it will be an indirect reference because of the inter-dll symbol resolution, and it needs to be exported otherwise the code will not link on windows (or with hidden symbols on linux).

@ghalliday - perhaps defaultFlushTimerInterval value should be copied into ActivityTimeAccumulator as a member, for speedier access?